modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2IDTokenEncryptionTestCase.java

```diff
@@ -119,15 +123,17 @@ public void testInit() throws Exception {
 
     @AfterClass(alwaysRun = true)
     public void atEnd() throws Exception {
-        deleteApplication();
-        removeOAuthApplicationData();
+        deleteApp(application.getId());
```

Comment:
Shall we fix the formatting issue by adding the missing new line?

Reply:
Fixed with https://github.com/wso2/product-is/pull/20096